pygmt/datasets/earth_relief.py

```diff
@@ -122,9 +138,14 @@ def load_earth_relief(resolution="01d", region=None, registration=None, use_srtm
         )
 
     # Choose earth relief data prefix
-    earth_relief_prefix = "earth_relief_"
-    if use_srtm and resolution in land_only_srtm_resolutions:
-        earth_relief_prefix = "srtm_relief_"
+    if data_source == "relief":
+        earth_relief_prefix = "earth_relief_"
+        if use_srtm and resolution in land_only_srtm_resolutions:
+            earth_relief_prefix = "srtm_relief_"
+    elif data_source == "gebco":
+        earth_relief_prefix = "earth_gebco_"
+    else:
+        raise GMTInvalidInput("Invalid input for 'data_source'.")
```

Comment:
Make the error a bit more informative.
```suggestion
        raise GMTInvalidInput(
            f"Invalid earth relief 'data_source' {data_source}, "
            "valid values are 'relief' and 'gebco'."
        )
```